cat-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<div class="container-fluid" id="wrapper">
+ 
+	<#include "${full_templates_path}/header/navigation.ftl" />
+
+    <#include "${full_templates_path}/header/hero.ftl" />
+
+    <main>
+
```

Comment:
We do not need a line break between the opening of a tag and the child element.  We only need aline break if it is a sibling element.